README.md

```diff
@@ -124,12 +124,38 @@ request.applePayDisabled = !canMakePayments
 
 ### 3D-Secure + Drop-In
 
-The new Drop-In supports 3D-Secure verification. If you have enabled 3D-Secure in the control panel, then just enable it in the BTDropInRequest and set an amount.
+The new Drop-In supports 3D-Secure verification. If you have enabled 3D-Secure in the control panel, enable it in the BTDropInRequest and set an amount. Then, create a BTThreeDSecureRequest() object, setting as many fields on it as possible; the more fields that are set, the less likely it is that a user will be be presented with a challenge. Make sure to attach this object to the BTDropInRequest before use.
 
 ```swift
-let request =  BTDropInRequest()
+let request = BTDropInRequest()
 request.threeDSecureVerification = true
-request.amount = "1.00"
```

Comment:
This change should be reverted, and the `BTDropInRequest.amount` should definitely not be deprecated. The majority of merchants likely aren't using 3DS, and the `amount` value here is what most of them will be setting.

Reply:
I agree that it shouldn't be marked deprecated, but this code snippet does live in the 3DS-specific section of the README, so it should probably use the most up-to-date 3DS conventions, no?

Reply:
Never mind, I didn't realize this value was only used for 3DS. It can probably be removed from the doc's then.

Reply:
Is the amount used for Apple Pay as well as 3ds?

Reply:
^ @Epreuve looked into this yesterday and couldn't find a conclusive answer; I think since the line isn't directly related to using 3DS 2.0, it's probably more important to get this merged than worry now (and we can always update it later).